ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/tls/DecryptionParameters.scala

```diff
@@ -46,7 +46,8 @@ case class DecryptionParameters(
     val cipher = Cipher.getInstance(transformation)
     val ivParameterSpec = new IvParameterSpec(iv)
     cipher.init(Cipher.DECRYPT_MODE, secretKey, ivParameterSpec)
-    cipher.doFinal(encrypted)
+    val digested = Try(Base64.getMimeDecoder.decode(encrypted)).getOrElse(encrypted)
```

Comment:
Is there even a remote possibility of an encrypted key looking as if it was base64 encoded even though it wasn't?

Reply:
Those files are 100+ characters long. There is a teeny tiny theoretical possibility for encoding to look like a base64.
Given that the universe is infinite, there is bound to be a developer somewhere who will be summoned to fix a bug for a customer who managed to generate such an encoding. In practice we can rest assure it will not happen.